File: src/Services/WareHouse/WareHouse.API/Application/Model/WareHouseBookDTO.cs
using System;

namespace WareHouse.API.Application.Model
{
    public class WareHouseBookDTO: BaseModel
    {
        public string VoucherCode { get; set; }
        public DateTime VoucherDate { get; set; }
        public string WareHouseId { get; set; }
        public string Deliver { get; set; }
        public string Receiver { get; set; }
        public string VendorId { get; set; }
        public string Reason { get; set; }
        public string ReasonDescription { get; set; }
        public string Description { get; set; }
        public string Reference { get; set; }
        public DateTime CreatedDate { get; set; }
        public string CreatedBy { get; set; }
        public DateTime ModifiedDate { get; set; }
        public string ModifiedBy { get; set; }
        public  string Type { get; set; }
        public  string WareHouseName { get; set; }
    }
}

File: src/WebClient/WareHouse/src/app/model/WareHouseBookDTO.ts
import { BaseModel } from "./BaseModel";


export interface WareHouseBookDTO extends BaseModel {
    voucherCode: string;
    voucherDate: Date;
    wareHouseId?: any;
    deliver: string;
    receiver: string;
    vendorId?: any;
    reason: string|null;
    reasonDescription?: any;
    description?: any;
    reference?: any;
    createdDate: Date;
    createdBy: string;
    modifiedDate: Date;
    modifiedBy?: any;
    type: string;
    wareHouseName:string |null;
}